Makefile

```diff
@@ -13,13 +13,15 @@ EXPORTER_NAME=zookeeper-exporter
 APP_NAME=zookeeper
 KUBE_VERSION=1.17.5
 REPO=pravega/$(PROJECT_NAME)
-TEST_REPO=$(DOCKER_USER)/$(PROJECT_NAME)
+TEST_REPO=testzkop/$(PROJECT_NAME)
 APP_REPO=pravega/$(APP_NAME)
 ALTREPO=emccorp/$(PROJECT_NAME)
 APP_ALTREPO=emccorp/$(APP_NAME)
 VERSION=$(shell git describe --always --tags --dirty | sed "s/\(.*\)-g`git rev-parse --short HEAD`/\1/")
 GIT_SHA=$(shell git rev-parse --short HEAD)
 TEST_IMAGE=$(TEST_REPO)-testimages:$(VERSION)
+DOCKER_TEST_PASS=testzkop@123
```

Comment:
I believe with this change we'll be able to run these tests on fork branches as well?

Reply:
Yes, these changes will run e2e test on fork branches as well.

Reply:
In that case, could you please add issue #181 in the "Fixes" section of this PR?

Reply:
sure, I have added it as well.